posthog/models/utils.py

```diff
@@ -154,7 +154,8 @@ class Meta:
 
 
 class DeletedMetaFields(models.Model):
-    deleted: models.BooleanField = models.BooleanField(null=True, blank=True)
+    deleted: models.BooleanField = models.BooleanField(null=True, blank=True, default=False)
+    deleted_at: models.DateTimeField = models.DateTimeField(null=True, blank=True)
```

Comment:
huh, how did this not trigger new migrations for all the models that use this mixin?

Reply:
Weirdly only warehouse tables use this mixin - all other models implement `deleted` field directly